app/views/admin/poll/polls/_form.html.erb

```diff
@@ -20,6 +20,22 @@
   </div>
 
   <div class="row">
+    <div class="small-12 column">
+      <%=f.text_area :summary, rows: 4%>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="small-12 column">
+      <%=f.text_area :description, rows: 8%>
```

Comment:
Same on this one, I fear we don't have a erb style checker to automate this corrections :( but a space after `<%=` and before `%>` (on previous comment as well) is always welcomed to make it easier to read 👍